docs/tutorial_quantum_sim/index.rst

```diff
@@ -0,0 +1,25 @@
+=============================
+Tutorial on quantum simulation
+=============================
+
+This experiment simulates the time evolution under a tight binding model on IBM Quantum hardware.
+The results are published at `arXiv:2209.02795`_.
+The documentation for this module consists of one notebook:
+
+1. The `qsim-tight-binding-model`_ contains background information, shows how the building blocks
+   of the experiment were implemented, collects experimental data, analyzes data, and generates plots.
+
+.. _arXiv:2209.02795: https://arxiv.org/abs/2209.02795
+.. _tutorial: qsim-tight-binding-model.ipynb
```

Comment:
It looks like the link is written incorrectly. Please build the docs locally and make sure they look correct.

Reply:
I've updated the index.rst files, and the building is doing better than before, but now it's timing out. I get this output:
```
.
.
.
Notebook error:
CellTimeoutError in tutorial_quantum_sim/qsim-tight-binding-model.ipynb:
A cell timed out while it was being executed, after 180 seconds.
The message was: Cell execution timed out.
Here is a preview of the cell contents:
-------------------
['# Evaulate all circuits over all time points and take the mean', '', '# Compute parameterized circuit over a range of trotter steps', 'trotter_steps_array = np.arange(1, 13)', '']
...
['    # Calculate average fidelity for a given trotter step', '    qc_rzx_fid_ests.append(np.mean(rzx_fids))', '', '    # Calculate average duration for a given trotter step', '    qc_rzx_dur_ests.append(np.mean(qc_dur))']
-------------------

Task was destroyed but it is pending!
task: <Task pending name='Task-453' coro=<NotebookClient._async_poll_output_msg() running at /Users/ajrasmusson/Documents/GitHub/qiskit-research/.tox/docs/lib/python3.8/site-packages/nbclient/client.py:812> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fe8b209b730>()]>>
ERROR: InvocationError for command /Users/ajrasmusson/Documents/GitHub/qiskit-research/.tox/docs/bin/sphinx-build -b html -W docs/ docs/_build/html (exited with code 2)
______________________________________________________ summary ______________________________________________________
ERROR:   docs: commands failed
```

I'm not sure what the error here is exactly. It seems like the tox -edocs is trying to run the notebook, but one of the cells is taking longer then 180 seconds?

Reply:
Please push your changes and I will investigate increasing the time limit. I'm wondering though, could you take a look at the long-running cell and think about whether there is a reasonable way to make it execute faster?